src/components/data/SelectionDrawer.js

```diff
@@ -170,6 +201,16 @@ function SelectionDrawer(props) {
             : false;
     };
 
+    const [toolbarHeight, setToolbarHeight] = useState(0);
+    const [appBarHeight] = useComponentHeight();
+
+    const toolbarHeightCallback = useCallback(
+        (height) => {
+            setToolbarHeight(height);
+        },
+        [setToolbarHeight]
+    );
```

Comment:
I believe this could be simplified since `useComponentHeight` does all the calculating for you.  So this `useState` and `useCallback` could turn into:
```
const [toolbarHeight, setToolbarRef] = useComponentHeight();
```

Reply:
This worked out. I still had to add offset for pagination toolbar but :thum

Reply:
Ah right, still need the pagination height factored in - good catch 👍